c/include/cuvs/neighbors/cagra.h

```diff
@@ -118,6 +120,45 @@ struct cuvsIvfPqParams {
 
 typedef struct cuvsIvfPqParams* cuvsIvfPqParams_t;
 
+/** Parameters for ACE (Augmented Core Extraction) graph build */
```

Comment:
I think this is a good opportunity to expand on the details behind the ACE algorithm. What is it? Under what conditions would a user want to choose this option? Maybe give a rough overview of what the algorithm does and any assumptions that it makes.